image_lists/globalAK.yml

```diff
@@ -0,0 +1,66 @@
+hourly:
```

Comment:
I see some diffs between image_lists for the 4 global ones. Should they be different? If not, then we may want to consider a different solution to how to handle the different tiles.

Reply:
They have been changed to match.  The only difference is the model names, which may need to be handled separately due to different projections.

Reply:
Since it's already happened at least once, this sort of large duplication will likely create big problems for us. I'd prefer to see this handled differently. Maybe I can try to get this knocked out in this "down week" so that we can also handle the RRFS domains similarly.

Reply:
I agree, there's a lot of duplication.  I would prefer to handle this in a separate PR.